index.html

```diff
@@ -0,0 +1,761 @@
+                                <span>Games</span>
+                            </li>
+                            <li class="boldItemSubNav topicList">
+                                <div class="iconContainer">
+                                    <svg class="navBottomIcon" viewBox="0 0 13 13" fill="#000"><path d="M12,2.9L9.6,5.2c-0.1,0.1-0.3,0.1-0.4,0C9.1,5.2,9.1,5,9.3,4.9l2.4-2.4c-0.2-0.2-0.3-0.3-0.5-0.5L8.7,4.3c-0.1,0.1-0.3,0.1-0.4,0C8.2,4.3,8.2,4.1,8.4,4l2.4-2.4c-0.3-0.3-0.5-0.5-0.5-0.5L7.6,3.4C7.1,4,6.8,5.1,7.1,5.8c-1.4,1-4.6,3.5-5.1,4c-0.8,0.8-0.4,1.8-0.3,1.9c0,0,0,0,0,0c0,0,0,0,0,0c0.1,0.1,1.1,0.5,1.9-0.3c0.4-0.4,2.9-3.6,3.9-5C8.4,6.9,9.6,6.6,10.2,6l2.3-2.6C12.5,3.4,12.3,3.2,12,2.9z"></path><path d="M0.8,1.9l0.3-0.3c0.9-0.9,3.2,1.1,3.8,1.7s0.9,1.8,0.4,2.6c1.4,1.1,4.6,3.5,5,3.9c0.8,0.8,0.4,1.8,0.3,1.9c0,0,0,0,0,0c0,0,0,0,0,0c-0.1,0.1-1.1,0.5-1.9-0.3c-0.4-0.4-2.9-3.7-4-5.1C3.9,6.7,2.9,6.4,2.3,5.8S-0.2,2.9,0.8,1.9z"></path></svg>
+                                    <span>Cooking</span>
+                                </div>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Email Newsletters</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Corporate Subscriptions</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Education Rate</span>
+                            </li>
+                        </ul>
+                        <ul class="itemsSubNavInfo topicList">
+                            <li class="normalItemSubNav topicList">
+                                <span>Mobile Application</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Replica Edition</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>International</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Canada</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Español</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>中文网</span>
```

Comment:
These items should be linked, using `#` as the value for their `href` attributes.